docs/DevGuide/Travis.md

```diff
@@ -75,6 +75,7 @@ if any of the following are true:
   - A `CMakeLists.txt` file in `src`, but not in an Executables or
     Python-binding directory,
     * does not list a `C++` file that is present in the directory
+    * lists a `C++` file that is not present in the directory
```

Comment:
I would have expected cmake to error if this is the case

Reply:
CMake errors on .cpp files, but doesn't care about missing headers.

The dependency checker does error for missing files, but we can't yet turn it on in CI. So we could get rid of this check once the dependency checker is run in CI?